gloo/test/multiproc_test.h

```diff
@@ -24,7 +24,7 @@ namespace gloo {
 namespace test {
 
 const int kExitWithIoException = 10;
-const auto kMultiProcTimeout = std::chrono::milliseconds(200);
+const auto kMultiProcTimeout = std::chrono::milliseconds(300);
```

Comment:
Does it hurt to make this a second or two to be conservative? Looks like this is the timeout for rendezvous.

Reply:
That's a reasonable point, let me bump it up a bit more to 500msec, and see what that does to test reliability. If it creeps up too high, my feeling is that any explicit timeout-type tests are going to take longer.

In a previous situation, we had an effective SLA that timing-related unittests on build machines should be resilient to a 100ms hiccup at any time - and seemed to make things fairly reliable.  This seems to get us there, and we can revisit if it's not sufficient.